ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml

```diff
@@ -1034,10 +1034,46 @@ keys:
                   - "discard"
               import_ethernet_segment_ip_mass_withdraw:
                 type: bool
+              import_overlay_index_gateway:
+                type: bool
               export_ethernet_segment_ip_mass_withdraw:
                 type: bool
           next_hop_unchanged:
             type: bool
+          bgp_additional_paths:
```

Comment:
@laxmikantchintakindi if this merges before your refactor PR this will need to be fixed as well

Reply:
Refactor PR is on hold till 5.0. I will consider this change for refactor.